Dockerfile

```diff
@@ -0,0 +1,39 @@
+# syntax=docker/dockerfile:1
+ARG GCC_VERSION=11.2.0
+ARG CMAKE_VERSION=3.22.0
+ARG GRPC_VERSION=1.43.0
+
+FROM iomz/cmake:$CMAKE_VERSION AS cmake
+FROM iomz/grpc-protoc:$GRPC_VERSION AS grpc
+
+# this stage builds xarm-grpc
+FROM gcc:$GCC_VERSION
```

Comment:
This is also nasty. Ideally, I want to just cross-compile the binary for alpine...